testconvolute.m

```diff
@@ -2,42 +2,52 @@
 ht = 0.01:0.01:2;%function h(t)
 Zt = convolute(xt,ht);%function Z(t), convoluted function
 cCheck = conv(0.01*xt,ht);%Our check for our test
-Zt %comment out if you want to print @MickeyDang
-cCheck %comment out if you want to print @MickeyDang
-T1 = false; %Bool for Test 1
-if is_appEq(Zt,cCheck) %Easier to set a tolerance because MATLAB has negative 0's
-    fprintf("Test1 has passed. Convolution matches with conv(xt,ht)");
-    T1 = true;
-end
+Zt; %comment out if you want to print @MickeyDang
+cCheck; %comment out if you want to print @MickeyDang
+T = false(2,1);
+T(1,1) = test_convolute(xt,ht,"1"); %Bool for Test 1
+
 
 %Run Test two 
-t = 0.01:0.01:2
+t = 0.01:0.01:2;
 sint = sin(2*pi*t);
 cost = cos(2*pi*t);
-Z2 = convolute(sint,cost);
-cCheck2 = conv(.01*sint,cost);
-Z2 %comment out if you want to print @MickeyDang
-cCheck2 %comment out if you want to print @MickeyDang
-T2 = false; %Bool for Test two
-if is_appEq(Z2,cCheck2)
-    fprintf("Test2 has passed. Convolution matches with conv(xt,ht)\n");
-    T2 = true;
-end
+T(2,1) = test_convolute(xt,ht,"2"); %Bool for Test two

```

Comment:
💯 !

Reply:
There's no need to comment with the bool for test two though. i think its pretty obvious.